src/goalServices/changeGoalStatus.ts

```diff
@@ -6,15 +7,25 @@ interface GoalStatusChangeParams {
   newStatus: string;
   reason: string;
   context: string;
+  transaction?: Sequelize.Transaction;
 }
 
 export default async function changeGoalStatus({
   goalId,
-  userId,
+  userId = 5,
   newStatus,
   reason,
   context,
+  transaction = null,
 }: GoalStatusChangeParams) {
+  // If userId is null, and we're in CI or a Jest test, just set it to 5.
+  // This let's us simplify the way we write the many tests that end up using this function
+  // one way or another, since mocking httpContext is a pain.
+  if (!userId && (process.env.NODE_ENV === 'test' || process.env.CI)) {
+    // eslint-disable-next-line no-param-reassign
+    userId = 5;
+  }
```

Comment:
I know not everybody will like this, but the alternative (at least the way I see it) is to mock httpContext (because that's what is used to get the userId) in every test that changes the status of a Goal, which would be a lot of additional code, compared to just three lines here. I think it's a reasonable compromise, but open to better solutions if there are any.